src/lib/gridster.component.ts

```diff
@@ -50,7 +57,25 @@ export class GridsterComponent implements OnInit, OnChanges, OnDestroy, Gridster
   emptyCell: GridsterEmptyCell;
   compact: GridsterCompact;
 
-  constructor(el: ElementRef, public renderer: Renderer2, public cdRef: ChangeDetectorRef, public zone: NgZone) {
+  @Output()
+  selectedItemChange = new EventEmitter<GridsterItem>();
+
+  private _selectedItem: GridsterItem;
+
+  @Input()
+  public get selectedItem(): GridsterItem {
+    return this._selectedItem;
+  }
+  public set selectedItem(v: GridsterItem) {
+    this._selectedItem = v;
+    this.selectedItemChange.emit(v);
+  }
+
+
+  @ViewChildren(forwardRef(() => GridsterItemComponent))
+  gridsterItems: QueryList<GridsterItemComponent>;
```

Comment:
why do you need this?

Reply:
was my first approach to get to the components. Dead code. Will be removed!